reference/halyard/high-availability.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: single
+title:  "High Availability"
+sidebar:
+  nav: reference
+---
+
+{% include toc %}
+
+This page describes how you can configure a Halyard deployment to increase the availability of specific services beyond simply [horizontally scaling](/setup/productionize/scaling/horizontal-scaling/) the service. Halyard does this by splitting the functionalities of a service into separate logical roles. The benefits of doing this is specific to the service that is being split. These deployment strategies are inspired by [Netflix's large scale experience](https://blog.spinnaker.io/scaling-spinnaker-at-netflix-part-1-8a5ae51ee6de).
+
+When split into different roles, the new logical services are given new names. This means that these logical services can be configured and scaled independently of each other.
```

Comment:
I agree that "roles" is confusing since it's usually only associated with authz. I don't mind calling these separate 'services', since technically each specialization of clouddriver/echo is a new service with its own config & load balancer.